Formula/presto.rb

```diff
@@ -12,7 +12,7 @@ class Presto < Formula
   resource "presto-cli" do
     version cli_version
     url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
```

Comment:
Some of it is in the [formula cookbook](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) (unfortunately not all). Some of it is enforced by the audits. Some of it is passed down by PR review.